docs/running.rst

```diff
@@ -317,6 +317,57 @@ There are several recommended ways to run Toil jobs in the cloud. Of these, runn
 
 On all cloud providers, it is recommended that you run long-running jobs on remote systems under ``screen``. Simply type ``screen`` to open a new ``screen` session. Later, type ``ctrl-a`` and then ``d`` to disconnect from it, and run ``screen -r`` to reconnect to it. Commands running under ``screen`` will continue running even when you are disconnected, allowing you to unplug your laptop and take it home without ending your Toil jobs.
 
+.. _Autoscaling:
+
+Autoscaling
+-----------
+The fastest way to get started runnning Toil in a cloud environment is using
+Toil's autoscaling capabilities to handle node provisioning for us.
+Currently, autoscaling is only supported on the AWS cloud platform with two
+choices of provisioners: Toil's own Docker based provisioner and CGCloud.
+
+The AWS provisioner is included in Toil alongside the [aws] extra and allows
+us to spin up a cluster without any external dependencies using the Toil
+Appliance docker image. To get started, we choose which version of the
+appliance to run from CGL's
+`Quay account <https://quay.io/repository/ucsc_cgl/toil-leader?tab=tags>`_and
+ set the TOIL_APPLIANCE_SELF environment variable to quay.io/ucsc-cgl:toil--
+ followed by the tag of the Toil Appliance version we wish to use ::
+
+    export TOIL_APPLIANCE_SELF=quay.io/ucsc_cgl/toil:3.5.0a1--80c340c5204bde016440e78e84350e3c13bd1801
+    _
+
+Having done that, simply run::
+
+    toil launch-cluster -p aws your-cluster-name-here --nodeType=t2.micro 
+    --keyPairName=your-AWS-key-pair-name
+    _
+
+to launch a t2.micro leader instance - you should adjust this instance type
+accordingly if you want to do real work. See 
+`here <https://aws.amazon.com/ec2/instance-types/>`_ for a full selection
+ of EC2 instance types. For more information on cluster managment using 
+ Toil's AWS provisioner, see :ref:`clusterRef`.
+
+To use CGCloud based autoscaling, see :ref:`installationAWS`for CGCloud
+installation and more information on starting our leader instance.
+
+Once we have our leader instance launched, the steps for both provisioners
+converge. At this point we simply start off our Toil run, being sure to
+set ``--batchSystem=mesos``, using an AWS job store with the following
+autoscaling specific options. Indicate your provisioner choice via the
+``--provisioner=<>`` flag and node type for your worker nodes via
+``--nodeType=<>``. Addionally, both provisioners support
+`preemptable nodes <https://aws.amazon.com/ec2/spot/>`_. Toil can run on a
+heterogenous cluster of both preemptable and non preemptable nodes. Our
+preemptable node type can be set by using the ``--preemptableNodeType=<>``
+flag. Be sure to read the help strings for this option, as the different
+provisioners have different syntax for indicating the bid price for on the
+node type you want. While individual jobs can each explicitly specify whether
```

Comment:
Yeah, I saw it in the help string here: https://github.com/BD2KGenomics/toil/blob/master/src/toil/common.py#L348
That's why the AWS provisioner uses c3.8xlarge:0.42 vs CGCloud's c3.8xlarge{|:0.42}